python/gaas_server/gaas_handler.py

```diff
@@ -279,7 +446,9 @@ def extract_subgraph(self,
                                     selection,
                                     edge_weight_property,
                                     default_edge_weight,
-                                    allow_multi_edges)
+                                    allow_multi_edges,
+                                    renumber_graph,
+                                    add_edge_data)
```

Comment:
Based on the arguments i guess this PR depends upton: 

https://github.com/rapidsai/cugraph/pull/2419/files

Reply:
it does